.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: Node CI
```

Comment:
```suggestion
name: CI
```

Reply:
It's node based, so...

Reply:
We also run shell scripts in here (https://github.com/cbitosc/cbitosc.github.io/pull/26/files#diff-60fe268f71ae7f8409aff40d0a3e51d8R25).